cityofthebigshoulders.js

```diff
@@ -916,6 +918,31 @@ function (dojo, declare) {
             }
         },
 
+        showFundraisingActions: function(round, phase)
+        {
+            // Building piles are placed over fundraising actions to hide them, but over decades they become visible.
+            var building_era = 0;
+            if (round >= 4 && phase >= 1)
+                building_era = 3;
+            else if (round == 3 || (round == 2 && phase >= 1))
+                building_era = 2;
+            else if (round >= 1 || phase >= 1)
+                building_era = 1;
+
+            // background-image = back of building card, so without that image it reveal the fundraising action in the boaard.
+            switch (building_era)
+            {
+                case 3:
+					dojo.style('fundraising_80', 'background-image', 'none');
+                case 2:
+					dojo.style('fundraising_60', 'background-image', 'none');
+                case 1:
+					dojo.style('fundraising_40', 'background-image', 'none');
+                case 0:
+                    break;
+            }
+        },
```

Comment:
This works, so I'll merge it as is.

Personally, I would've preferred doing it without a switch case with pass through cases.

e.g. if(round >= 1 && phase >= 1) then remove background for $40

Reply:
True, much better that now (I fixed too)